14_number_shuffle.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+# Implementation for RubyMonk Primer Problem 14 Number shuffle
+# Problem Statement:
+# Given a 3 or 4 digit number with distinct digits, return a sorted array
+# of all the unique numbers that can be formed with those digits.
+# Example:
+# Given: 123
+# Return: [123, 132, 213, 231, 312, 321]
+class NumberShuffle
+  def number_shuffle(number)
+    # your code here
+    perms_as_arr_of_strings = number.to_s.split('').permutation.to_a
```

Comment:
I dislike this variable name; I don't find it helpful.  Why do I care that this is an array of strings?

Reply:
Yeah, that `perms_as_arr_of_strings` was an attempt to break it up so that the line did not get too long which is a
contrivance and I think that that name is exposing this contrivance.  Good teaching moment on using `chars` 😍 .

Reply:
✅